docs/streams/developer-guide.html

```diff
@@ -267,21 +295,532 @@
+	      match the configured default serdes.</li>
+	  </ul>
+	  
+	  Several variants of <code>table</code> exist to e.g. specify the <code>auto.offset.reset</code>
+	  policy to be used when reading from the input topic.
+	</td>
+      <tr>
+        <td><b>Global Table</b>: input topic -> <code>GlobalKTable</code></td>
+	<td>
+	  Reads the specified Kafka input topic into a <code>GlobalKTable</code>. The topic is interpreted as a changelog stream, where records
+	  with the same key are interpreted as UPSERT aka INSERT/UPDATE (when the record value is not <code>null</code>) or as DELETE (when the
+	  value is <code>null</code>) for that key. 
+	  <p>
+	    Slightly simplified, in the case of a GlobalKTable, the local GlobalKTable instance of every application instance will be
+	    populated with data from all the partitions of the input topic. In other words, when using a global table, every application
+	    instance will get its own, full copy of the topic’s data.
+
+	  <p>
+	    You must provide a name for the table (more precisely, for the internal state store that backs the table).
+	    This is required, among other things, for supporting interactive queries against the table.
+
+	    <pre class="brush: java;">
+        import org.apache.kafka.common.serialization.Serdes;
+        import org.apache.kafka.streams.kstream.KStreamBuilder;
+        import org.apache.kafka.streams.kstream.GlobalKTable;
+
+	        KStreamBuilder builder = new KStreamBuilder();
+
+	        GlobalKTable&lt;String, Long&gt; wordCounts = builder.globalTable(
+		        Serdes.String(), /* key serde */
+		        Serdes.Long(),   /* value serde */
+		        "word-counts-input-topic", /* input topic */
+		        "word-counts-global-store" /* table/store name */);
+	    </pre>
+	    
+	    When to provide serdes explicitly:
+	    <ul>
+	    <li>If you do not specify serdes explicitly, the default serdes from the configuration are used.</li>
+	    <li>You must specificy serdes explicitly if the key and/or value types of the records in the Kafka input topic do not
+	      match the configured default serdes.</li>
+	    
+	    Several variants of globalTable exist to e.g. specify explicit serdes.
+
+	</td>
+  </tbody>
+  </table>
 
-    KStream&lt;String, GenericRecord&gt; source1 = builder.stream("topic1", "topic2");
-    KTable&lt;String, GenericRecord&gt; source2 = builder.table("topic3", "stateStoreName");
-    GlobalKTable&lt;String, GenericRecord&gt; source2 = builder.globalTable("topic4", "globalStoreName");
-    </pre>
+    <h4><a id="streams_dsl_transform" href="#streams_dsl_transform">Transform a stream</a></h4>
 
-    <h4><a id="streams_dsl_windowing" href="#streams_dsl_windowing">Windowing a stream</a></h4>
+    <code>KStream</code> and <code>KTable</code> support a variety of transformation operations. Each of these operations can be translated into one or more
```

Comment:
This paragraph needs to be wrapped into a `<p>` or sth -- it displayed as a single paragraph together with the one after.